pkg/deploy/controller/deployment/factory.go

```diff
@@ -28,8 +28,10 @@ type DeploymentControllerFactory struct {
 	Codec runtime.Codec
 	// Environment is a set of environment which should be injected into all deployer pod containers.
 	Environment []kapi.EnvVar
-	// RecreateStrategyImage specifies which Docker image which should implement the Recreate strategy.
+	// RecreateStrategyImage specifies which Docker image should implement the Recreate strategy.
 	RecreateStrategyImage string
+	// RollingStrategyImage specified which Docker image should implement the Rolling strategy.
+	RollingStrategyImage string
```

Comment:
Let's not have one image per strategy.  I don't see the need for the config anytime soon - there's just a deployer image that supports multiple strategies.